tests/auth_tests/test_forms.py

```diff
@@ -1201,6 +1222,7 @@ def test_success(self, password_changed):
         self.assertEqual(password_changed.call_count, 0)
         form.save()
         self.assertEqual(password_changed.call_count, 1)
+        self.assertEqual(form.changed_data, ["password"])
```

Comment:
Just curious what was the rationale for including these assertions on changed_data?

Reply:
We are on the same page shangxiao!

At first, I just added the assertion to cover the lines inside of the `changed_data` property (as I show in the third screenshot) 

I'm still a bit confused about the use/proposed of the property but then, when I added to every `test_` inside of `AdminPasswordChangeFormTest` I realized that the prop returns `['password']` when both fields were completed, if one or two of them are empty/None it will return `[]`